app/controllers/api/v0/events_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Api::V0::EventsController < ApplicationController
+  def create
+    KafkaClient.produce(data: params[:event][:data], topic: params[:event][:topic])
```

Comment:
do you have a plan in your head for something other than this `event` wrapper at the top level? we might want to just get rid of that

Reply:
@TomWoodward for sure i did...i figured that the KafkaClient will encapsulate a lot of stuff down the road, like schema validations, logging, exception handling, and prob. more.